test/spec/MessagesSpec.js

```
@@ -13,23 +13,22 @@ import {
   changeInput,
-  findDivByClass
 } from './helpers';
 import spiffModdleExtension from '../../app/spiffworkflow/moddle/spiffworkflow.json';
 import messages from '../../app/spiffworkflow/messages';
 import { fireEvent } from '@testing-library/preact';
 import { getBpmnJS, inject } from 'bpmn-js/test/helper';
 import { findCorrelationProperties, findMessageModdleElements } from '../../app/spiffworkflow/messages/MessageHelpers';
 
+import {
+  spiffExtensionOptions,
+} from '../../app/spiffworkflow/extensions/propertiesPanel/SpiffExtensionSelect';
```

ESLint

[error] Replace `⏎··spiffExtensionOptions,⏎` with `·spiffExtensionOptions·`

(prettier/prettier)